content-plugins.md

```diff
@@ -7,7 +7,10 @@ toc: true
 
 ## Content Plugins
 
-After you install Pulp, you must add a content plugin for each content type that you want to manage. The following sections contain information about the available content plugins for both Pulp 2 and Pulp 3. If you do not find the plugin for the content type you want to manage, consider [writing a plugin](https://docs.pulpproject.org/plugins/plugin-writer/index.html).
+After you install Pulp, you must add a content plugin for each content type that you want to manage.
+The following sections contain information about the available content plugins for both Pulp 2 and
+Pulp 3. If you do not find the plugin for the content type you want to manage, consider
+[writing a plugin](https://docs.pulpproject.org/plugins/plugin-writer/index.html).
```

Comment:
I'd say, whatever you do, picking plugins to install is one of the first steps in the installation process.

Reply:
Good point!